src/actions/account.js

```diff
@@ -248,3 +255,19 @@ export const accountDataUpdated = ({
       dispatch(liskAPIClientUpdate({ online: false, code: res.error.code }));
     });
   };
+
+export const updateAccountDelegateStats = account =>
+  (dispatch, getState) => {
+    const liskAPIClient = getState().peers.liskAPIClient;
+    const { address, publicKey } = account;
+    getTransactions({
```

Comment:
man there is no problem here with nothing everything works fine, is just that after check this with @slaweet every new code that we create based on promises we need to do it with Async/Await, so you dont need to change much but yes please change it to async/await